spec/factories/people.rb

```diff
@@ -1,6 +1,8 @@
 FactoryGirl.define do
   factory :person do
-    name { Faker::Name.name }
+    division
+    first_name { Faker::Name.name }
```

Comment:
`Faker::Name` has `.first_name` and `.last_name` methods for this. `.name` will generate full names including occasional titles/qualifiers.